gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-edit-page/api-documentation-v4-edit-page.component.ts

```diff
@@ -73,6 +75,8 @@ export class ApiDocumentationV4EditPageComponent implements OnInit, OnDestroy {
       content: this.formBuilder.control('', [Validators.required]),
     });
 
+    this.pageType = this.activatedRoute.snapshot.queryParams.pageType;
```

Comment:
This isn't necessary because the page type will be resolved by the returned page, right?

Reply:
pageType is used in the template to decide what to display so we need to initialize it with the proper value. And we need to handle page creation and page edition. So we need to keep it.